packages/itwin/tree-widget/src/components/trees/category-tree/CategoriesTreeComponent.tsx

```diff
@@ -27,38 +29,99 @@ export function CategoriesTreeComponent(props: CategoriesTreeProps) {
     filteredProvider,
   } = useTreeFilteringState();
 
+  return (
+    <>
+      {iModel && viewport &&
+        <>
+          <TreeHeaderComponent
+            searchOptions={searchOptions}
+            treeHeaderButtons={props.TreeHeaderButtons
+              ? props.TreeHeaderButtons.map((btn) => btn({ iModel, viewport, filteredProvider }))
+              : [
+                <ShowAllButtonCategoriesTree iModel={iModel} filteredProvider={filteredProvider} key="show-all-btn" />,
+                <HideAllButtonCategoriesTree iModel={iModel} filteredProvider={filteredProvider} key="hide-all-btn" />,
+                <InvertButtonCategoriesTree iModel={iModel} viewport={viewport} filteredProvider={filteredProvider} key="invert-all-btn" />,
```

Comment:
Maybe move all the buttons under `CategoriesTreeComponent` namespace?

Reply:
fixed

Reply:
My idea was to move just the buttons and name the namespace with the same name as the component (`CategoriesTreeComponent`). But I like @saskliutas suggestion even better: https://github.com/iTwin/viewer-components-react/pull/396#discussion_r1139900293.

Reply:
My idea was to move only the buttons, and name the namespace similar to the component - `CategoriesTreeComponent`. But I like @saskliutas suggestion even better: https://github.com/iTwin/viewer-components-react/pull/396#discussion_r1139900293